content/docs/sidebar.json

```diff
@@ -524,33 +506,11 @@
     "type": "external"
   },
   {
-    "label": "CML",
     "slug": "cml",
-    "source": "cml/index.md",
+    "label": "CML",
     "icon": "cml",
-    "children": [
```

Comment:
`cml` -> "CML" would also be good to auto-capitalize. Plus how is this external link implemented now? The nav still has a substructure. Seems like it's hard-coded somewhere? We can remove that 🙂 cc @rogermparent thanks

Reply:
If I'm understanding correctly, we have a line in [`redirects.json`](https://github.com/iterative/dvc.org/blob/master/redirects-list.json#L78) that converts the `/doc/cml` to `https://cml.dev/doc`.